nixos/modules/services/development/corteza.nix

```diff
@@ -0,0 +1,115 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.corteza;
+in
+{
+  options.services.corteza = {
+    enable = lib.mkEnableOption "Corteza, a low-code platform";
+    package = lib.mkPackageOption pkgs "corteza" { };
+
+    address = lib.mkOption {
+      type = lib.types.str;
+      default = "0.0.0.0";
+      description = ''
+        IP for the HTTP server.
+      '';
+    };
+    port = lib.mkOption {
+      type = lib.types.port;
+      default = 80;
+      description = ''
+        Port for the HTTP server.
+      '';
+    };
+    openFirewall = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
+      example = true;
+      description = "Whether to open ports in the firewall.";
+    };
+
+    user = lib.mkOption {
+      type = lib.types.str;
+      default = "corteza";
+      description = "The user to run Corteza under.";
+    };
+
+    group = lib.mkOption {
+      type = lib.types.str;
+      default = "corteza";
+      description = "The group to run Corteza under.";
+    };
+
+    settings = lib.mkOption {
+      type = lib.types.submodule {
+        freeformType = lib.types.attrsOf lib.types.str;
+        options = {
+          HTTP_WEBAPP_ENABLED = lib.mkEnableOption "webapps" // {
+            default = true;
+            apply = toString;
+          };
+        };
+      };
+      default = { };
+      description = ''
+        Configuration for Corteza, will be passed as environment variables.
+        See <https://docs.cortezaproject.org/corteza-docs/2024.9/devops-guide/references/configuration/server.html>.
+      '';
+    };
+
+    database.createLocally = lib.mkEnableOption "configuring a local PostgreSQL database";
```

Comment:
Let's remove this since it's not actually creating the database. I think the warning should be good enough for now, but let's discuss this in more detail later on.